libs/shared/ui/src/lib/hero-banner-util-button/hero-banner-util-button.component.ts

```diff
@@ -1,11 +1,13 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'biosimulations-hero-banner-util-button',
   templateUrl: './hero-banner-util-button.component.html',
   styleUrls: ['./hero-banner-util-button.component.scss'],
 })
 export class HeroBannerUtilButtonComponent {
+  @Input()
+  public appName!: string;
```

Comment:
This is probably out of scope and unimportant here - just a comment about style and type safety.  I'm not asking you to make the following change - but consider it for the future.

I'm not crazy about appName being a string with string literals in the corresponding HTML (if we change app names or misspell).  I was thinking about typing this as a union of literals (e.g. "BioSimulators" | "BioSimulations" | ...).

Potential change

in config.ts (or wherever the config typing is defined):
`type AppName = "BioSimulators" | "BioSimulations" | ..`. 
`appName: AppName;`

then here appName string compares will be checked by the compiler (I think)
`appName: AppName;`

Reply:
I like your approach of having a mapping rather than a metaphysically ternary operation: we currently have no way of validating the string literal in the above change and original implementation.